src/reducers/main-menu.js

```diff
@@ -17,36 +17,36 @@ const defaultState = {
       label: 'OVERVIEW',
       attrs: {
         tracking: 'dashboard',
-        href: 'https://kat.ft.com/overview'
+        href: '/overview'
       }
     },
     {
       label: 'CONTENT DISTRIBUTION',
       attrs: {
         tracking: 'myFT',
-        href: 'https://kat.ft.com/myft'
+        href: '/myft'
       }
     },
     {
       label: 'USAGE REPORTS',
       attrs: {
         tracking: 'report',
-        href: 'https://kat.ft.com/usage'
+        href: '/usage'
       }
     },
     {
       label: 'USER MANAGEMENT',
       attrs: {
         tracking: 'users',
-        href: 'https://kat.ft.com/users'
+        href: '/users'
```

Comment:
Probably need to arrange a redirect for this to licence admin until our users app is live. V1 has `Licence Administration` instead but it would be good to get the apps using V1 bumped to V2 as soon as we can.

Reply:
I don't think we show the `users` tab to anyone yet. AFAIK it's behind a flag. In the meantime we have the `Licence Administration` tab which _does_ go there